test/ooni/utils.cpp

```diff
@@ -30,6 +30,12 @@ TEST_CASE("geoip works") {
     REQUIRE(((*json)["country_name"] == std::string{"United States"}));
 }
 
+TEST_CASE("geoip can handle an invalid database") {
```

Comment:
As above, it's not invalid rather you cannot open it (I think invalid is a narrower concept)